tests/test_store.py

```diff
@@ -889,51 +896,62 @@ class TestTableOperations:
         ["display_label", "class_label"],
         ids=["aks_display_label", "aks_class_label"],
     )
-    def test_createTable(
+    def test_create_table(
```

Comment:
The TLDR of these changes:

**Problem:**
1. When a test is executed, it uses a common synapse resource by name and parent.
2. More than a single test was executing at any single point in time (In the same python version, across python versions, and across branches)


Because multiple resources were pointing to the same Synapse resources it would cause common 404 (Not found), or 412 (Conflicts) errors.


**Solution, I have updated the logic here to:**

1. Copy the synapse resources in question. Any folders being used are copied under a unique `datasetId` folder.
2. Tables have unique names with UUIDs.


**Some notes:**

1. If an integration test bails before the resource can be cleaned up we can be left with a bunch of resources leftover in the test project. If this becomes an issue we can write a small script that walks these projects and destroys resources that have `integration_test_` in the name.